include/in_process_server.h

```diff
@@ -201,6 +211,9 @@ class Client
     Surface create_xdg_shell_stable_surface(int width, int height);
     Surface create_visible_surface(int width, int height);
 
+    size_t output_count() const;
+    OutputState output_state(size_t index) const;
```

Comment:
I wish it were less of an absolute pain in the arse to provide iterators in C++. This is an crying out to be an iterator.